schema.cc

```diff
@@ -437,6 +437,22 @@ schema::schema(const schema& o)
 {
 }
 
+schema::schema(reversed_tag, const schema& o)
+    : schema(o, [] (schema& s) {
+        // We invert the first 32bit octet of the MSB of the UUID, so doing two reversals gets us back to the original UUID
+        const int64_t mask = 0x00000000FFFFFFFF;
+        auto msb = s._raw._version.get_most_significant_bits();
```

Comment:
Another problem here is that not all uuids are time uuids here. schema_mutations::digest() generates name uuids.